### PR TITLE
Remove Walmart med price pilot

### PR DESCRIPTION
**Pull request by jknapp25**
Now that we have enough data around surfacing med price for walmart, and given it didn't change conversion, there's no need to keep this around.

### CODE REVIEW

File: apps/patient/src/components/PharmacyInfo.tsx [hunk 1]
```diff
@@ -90,9 +89,7 @@ export const PharmacyInfo = ({
   preferred = false,
   showDetails = true,
   boldPharmacyName = true,
-  isStatus = false,
-  price = null,
-  selected = false
```
Comment (marisbest2) [inline apps/patient/src/components/PharmacyInfo.tsx:95]: Do we not need the selected?

File: apps/patient/src/utils/general.ts [hunk 2]
```diff
@@ -7,11 +7,9 @@ import timezone from 'dayjs/plugin/timezone';
 import utc from 'dayjs/plugin/utc';
 import costcoLogo from '../assets/costco_small.png';
 import walgreensLogo from '../assets/walgreens_small.png';
-import walmartLogo from '../assets/walmart_small.svg';
```
Comment (marisbest2) [inline apps/patient/src/utils/general.ts:10]: Do we want to keep the Walmart logo? I get that price didnt make a difference but I feel like it's nice to be able to see the logo?

File: apps/patient/src/components/PharmacyInfo.tsx (hunk 1 above)
Comment (jknapp25) [reply apps/patient/src/components/PharmacyInfo.tsx:95]: it was only being used to surface the cash price disclaimer, now that that's getting removed, there's no reason for that prop

File: apps/patient/src/utils/general.ts (hunk 2 above)
Comment (captDaylight) [reply apps/patient/src/utils/general.ts:10]: was thinking the same thing, but maybe this is something we don't give away for free to a pharmacy? But it is a nicer looking interface

Comment (marisbest2) [reply apps/patient/src/utils/general.ts:10]: Yeah that makes sense to me. Idk

**Review by marisbest2 — APPROVED** [review]

**Pull request merged**